core/dataset.cc

```diff
@@ -1224,7 +1228,7 @@ queryBasic(const SqlBindingScope & scope,
            bool allowParallel) const
 {
     // 1.  Get the rows that match the where clause
-    auto rowGenerator = generateRowsWhere(scope, where, 0 /* offset */, -1 /* limit */);
+    auto rowGenerator = generateRowsWhere(scope, "", where, 0 /* offset */, -1 /* limit */);
```

Comment:
Consider adding /* alias */ so that it is easier to know what this empty string is referring to.